src/RecoveryServices/RecoveryServices.SiteRecovery/Models/PSObjects.cs

```diff
@@ -2340,6 +2340,24 @@ public class ASRVMNicConfig
         [DataMember]
         public string RecoveryVMNetworkId { get; set; }
 
+        /// <summary>
```

Comment:
Also, since changes are required to an existing cmdlet, I would suggest you go ahead and raise a design issue first (take a look at - [https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/460](https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/460)).

Once that's approved, you can share the link on this PR and proceed with the merge

Reply:
ok